CMakeLists.txt

```diff
@@ -54,6 +54,12 @@ if(CMAKE_BUILD_TYPE STREQUAL "Debug")
     set(IS_DEBUG_BUILD ON)
 endif()
 
+option(HIDAPI_ENABLE_ASAN "Build HIDAPI with ASAN address sanitizer instrumentation" ON)
```

Comment:
I believe this is a bit too aggressive.
We probably want to set it to `OFF` by default, and explicitly enable it in CI scripts for `TEST` version of the builds.

And no, we don't have a `TEST` versions of the build yet - those needs to be introduced, at least for Windows, where we run unit-tests.